lib/graphql/schema_comparator/changes.rb

```diff
@@ -215,7 +215,27 @@ def message
         end
 
         def path
-          # TODO
+          new_schema.query.graphql_name
+        end
+      end
+
+      class SchemaQueryTypeAdded < AbstractChange
+        attr_reader :old_schema, :new_schema, :criticality
+
+        def initialize(old_schema, new_schema)
+          @old_schema = old_schema
+          @new_schema = new_schema
+          @criticality = Changes::Criticality.non_breaking(
+            reason: "Adding a schema query root is considered non-breaking."
+          )
+        end
+
+        def message
+          "Schema query root `#{new_schema.query&.graphql_name}` was added."
+        end
+
+        def path
+          new_schema.query&.graphql_name
```

Comment:
If this change existed, wouldn't `new_schema.query` be guaranteed to exist?

Reply:
Yup, good catch 👍🏻